proto/api/v1/api.proto

```diff
@@ -570,6 +576,17 @@ message CreateConnectorResponse {
   Connector connector = 1;
 }
 
+message ValidateConnectorRequest {
+  Connector.Type type = 1;
+  // Plugin name is the name of the builtin plugin (builtin:name), or the absolute path of a standalone plugin.
+  string plugin = 2;
+  // Configurations for the connector to get validated
+  Connector.Config config = 4;
+}
+
+message ValidateConnectorResponse {
+}
```

Comment:
Do we need an error here?

Reply:
at first, I had an error here, but then decided to go with returning the error, which would look like:
`{
    "code": 13,
    "message": "configurations failed: \"path\" config value must be set",
    "details": []
}` 
but If having the error as a string in the response makes more sense, then I can go with that.. WDYT?

Reply:
Given that, it's fine IMO. :+1: